tests/plugins/pastefromwordimage/manual/pastewordwithimage.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: bug, 4.8.0, 662, pastefromwordimage
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, pastefromwordimage, sourcearea, elementspath, newpage, resize
+
+----
+
+1. Open document and copy whole content.
+1. Clear editor content.
+1. Paste copied content into CKEditor.
+
+### Perform above steps for all files:
+  * [Offline img](../generated/_fixtures/SimpleImages/SimpleImages.docx).
+  * [Online and offline img](../generated/_fixtures/MixedOnline/Mixed_drawings_and_online.docx),
+  * [Document with shape](../generated/_fixtures/MixedOnlineAndShapes/Mixed_drawings_and_online_and_shapes.docx). _Please notice that shape should be ignored while pasting._
+
+**Expected:** Images are pasted to the editor.
```

Comment:
As one of the files contains a shape which is not pasted it should ba also described here, e.g. _Keep in mind that shapes (like last object in`Mixed_drawings_and_online_and_shapes.docx`) will not be pasted_.